src/main/java/seedu/address/logic/commands/FindStarsLessThanCommand.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.person.StarsLessThanPredicate;
+
+/**
+ * Finds and lists all persons in address book whose number of stars is less than the argument.
+ * Comparator is strictly less than.
+ */
+public class FindStarsLessThanCommand extends Command {
+    public static final String COMMAND_WORD = "findStarsLessThan";
```

Comment:
might be good to think of a shorter command phrase.

Reply:
I suggest findSLT? or findStarLT?

Reply:
Changed to findStarsLT